src/WebJobs.Script.Scaling/AzureScaleHandler.cs

```diff
@@ -0,0 +1,287 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Scaling
+{
+    public sealed class AzureScaleHandler : IScaleHandler
```

Comment:
I wonder if these type names shouldn't have prefix "AppService" rather than "Azure" since this is AppService specific. E.g. "AppServiceScaleHandler", or even better AppServiceScaleManager